pymatgen/io/lobster/inputs.py

```diff
@@ -82,8 +81,8 @@ class Lobsterin(UserDict, MSONable):
         "kSpaceCOHP",
         "EwaldSum",
     )
-    # the keyword alone will turn on or off a function
-    BOOLEAN_KEYWORDS = (
+    # The keywords alone will turn on or off a function
```

Comment:
```suggestion
    # These keywords can trigger additional functionalities
```

Reply:
I think the previous expression is already clear enough for this one... And I would like to hear about other comments too :)

Reply:
I found the "alone" part strange and "turn on or off" a somewhat cumbersome expression.

Reply:
> I found the "alone" part strange and "turn on or off" a somewhat cumbersome expression.

I guess `along` refers to no additional arguments (just the command), for example `skipdos` vs `COHPstartEnergy -35`, yes the `turn on or off` sounds a bit too verbose

Reply:
I know what the "alone" was referring to. I'm just thinking of a way to say this more elegant and efficient.

Reply:
Yes I would trust you as the language expert :) Perhaps asking LLMs

Reply:
```suggestion
    # The keywords themselves (without suffix) can trigger additional functionalities.
```

Reply:
Thanks!